Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM quay.io/bitnami/golang:1.16 as builder
+FROM golang:1.16 as builder
```

Comment:
(and by user, I meant the one building it)

Reply:
How about using the makefile to pass the options and changing the Dockefile to something like  `FROM $IMG_NAME:$IMG_VER` ?
The default can be set to quay.io to avoid hitting the docker hub rate-limiting issue.

Reply:
See update for suggestion. Podman will yell for using defaults, I don't know if that's a problem for us:
```
WARN[0000] missing "IMG_NAME" build argument. Try adding "--build-arg IMG_NAME=<VALUE>" to the command line
```
Btw, I'm doing something similar in the sort-of-related https://github.com/kata-containers/kata-containers/pull/3724.